hack/install-verify-tools.sh

```diff
@@ -18,7 +18,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+"${KUBE_ROOT}"/hack/check-go-version.sh
```

Comment:
I think it would be good not to mix functional changes with formatting/code fixes.